README.md

```diff
@@ -95,7 +97,12 @@ This'll initialize text field component on a single `.mdc-text-field` element.
 
 > Please see [quick start demo](https://stackblitz.com/edit/mdc-web-quick-start-demo) on stackblitz for full example.
 
-> Please see [getting started guide](docs/getting-started.md) for pointers on how to configure webpack to bundle JavaScript & CSS.
+
+## Examples using MDC Web
```

Comment:
Just a thought, maybe it makes sense to put this in a separate doc which just shows list of all examples if we're planning to add more later. Idea is to keep quick start guide getting started less overwhelming. WDYT?

Reply:
Sorry for the major delay! moved the examples to a separate doc linked from the main README.